Realm.Shared/RealmObject.cs

```diff
@@ -43,6 +43,13 @@ public class RealmObject
         /// </summary>
         public bool IsManaged => _realm != null;
 
+        /// <summary>
+        /// Returns true if this object is managed and represents a row in the database.
+        /// If the object has been deleted (say, on another thread), it is no longer valid and accessing properties on it
```

Comment:
The aside that mentions another thread is a little weird too. It may give users the impression that objects can be invalidated at any time, when in fact they'll only be invalidated by another thread when the Realm they belong to is advanced to a newer version.